src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -408,6 +408,12 @@ static void read_channel_args(grpc_chttp2_transport* t,
                         ? grpc_core::Duration::Infinity()
                         : (t->is_client ? g_default_client_keepalive_timeout
                                         : g_default_server_keepalive_timeout)));
+  t->ping_timeout = std::max(
+      grpc_core::Duration::Zero(),
```

Comment:
zero duration?

Reply:
same as keepalive_timeout... we should maybe choose a safer minimum